include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -605,7 +605,7 @@ __pattern_minmax_element(__hetero_tag<_BackendTag>, _ExecutionPolicy&& __exec, _
 
     // This operator doesn't track the lowest found index in case of equal min. values and the highest found index in
     // case of equal max. values. Thus, this operator is not commutative.
-    __pattern_minmax_element__reduce_fn<_Compare, _ReduceValueType> __reduce_fn{__comp};
+    __pattern_minmax_element_reduce_fn<_Compare, _ReduceValueType> __reduce_fn{__comp};
```

Comment:
Seems to be an unrelated change.

Reply:
Also, it was the comment from @adamfidel 
I think this small change make sense.
It's just incorrect structure name from one of my previous PRs.
I propose to save as is.

Reply:
Probably earlier before one of mine PRs they were duplicated lambdas.
I propose to fix it in separate PR and just create issue now for that.

Reply:
Or I can move the into `utils_hetero.h` file in this PR...

Reply:
I agree, it should be an issue, and perhaps we can address this naming thing when we remove these redundant callables.  We need to take a look through #2153, because it seems there are more than just the ones mentioned.

Reply:
Reverted

Reply:
Issue created: https://github.com/uxlfoundation/oneDPL/issues/2199

Reply:
PR created: https://github.com/uxlfoundation/oneDPL/pull/2200

Reply:
PR created: https://github.com/uxlfoundation/oneDPL/pull/2200 has been merged into main branch, so I resolve this conversation.